### ISSUE TITLE
The left file panel is too dark

### ISSUE THREAD

**Issue by liuxsen** [wesbos/cobalt2-vscode#214]
<img width="290" alt="image" src="https://user-images.githubusercontent.com/12598696/197244591-faa0a919-4185-44a8-ad9a-bd1e16f28dae.png">

I think the color contrast should be increased
